target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -222,6 +222,7 @@ ath79_setup_interfaces()
 	dlink,dir-842-c2|\
 	dlink,dir-842-c3|\
 	dlink,dir-859-a1|\
+	dlink,dir-869-a1|\
```

Comment:
And do not forget set mac address in this file. Just like the dir-859.

Reply:
Try to use `mtd_get_mac_text` instead of `mtd_get_mac_ascii` so that we can easily set mac address in dts after we convert to DSA driver. `devdata` seems not a writable partition, so I believe mac address offset are fixed.

Reply:
Yes, you should separate them. And you also need to find the mac address offset, refer to:
https://github.com/openwrt/openwrt/blob/80a62a675d2ef57a1885e1829926dd2ea1682612/target/linux/ath79/generic/base-files/etc/board.d/02_network#L609
If you feel difficult about it, then you can upload the `devdata` partition here, I can help to analyze it.

Reply:
If I understand it correctly, the offset is 0xc9. I am sending devdata.
[dir-869.mtd2.bin.zip](https://github.com/openwrt/openwrt/files/9863281/dir-869.mtd2.bin.zip)

Reply:
> If I understand it correctly, the offset is 0xc9 for lan_mac and 0x79 for wan_mac.

Yes you are right.
Considering that it has been more than one year since you create this PR, would you mind rebase it to current master branch in case we miss some important things?

Reply:
I got a flash backup of dir-859 A3. It seems that their mac addresses have different offsets. Unfortunately, we have to revert these changes.
![image](https://user-images.githubusercontent.com/70847398/201042255-badcd9dd-a828-4e64-9a8a-4323313d5955.png)

Reply:
Hi @DragonBluep, can you please send your offset so I can split the devices in the 02_network file?

Reply:
I think this format should use the `mtd_get_mac_ascii` function to parse the equals-separated values, even though this is currently not supported to be set within dts. But technically, the offsets might change even between two devices of the same series (e.g. depending on the exact OEM firmware version that changed).